awx/main/tests/unit/conf/test_fields.py

```diff
@@ -0,0 +1,71 @@
+import pytest
```

Comment:
There's `awx/mainconf/tests/unit/` available for hosting Tower Configuration tests.